content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -460,14 +460,25 @@ An example flow:
       The containers in the Pod receive the TERM signal at different times and in an arbitrary
       order. If the order of shutdowns matters, consider using a `preStop` hook to synchronize.
       {{< /note >}}
-1. At the same time as the kubelet is starting graceful shutdown, the control plane removes that
-   shutting-down Pod from Endpoints (and, if enabled, EndpointSlice) objects where these represent
+
+1. At the same time as the kubelet is starting graceful shutdown, the control plane evaluates
+   whether a terminating Pod needs to be removed from any Endpoints and if enabled,
+   EndpointSlice objects where these represent
    a {{< glossary_tooltip term_id="service" text="Service" >}} with a configured
    {{< glossary_tooltip text="selector" term_id="selector" >}}.
    {{< glossary_tooltip text="ReplicaSets" term_id="replica-set" >}} and other workload resources
    no longer treat the shutting-down Pod as a valid, in-service replica. Pods that shut down slowly
-   cannot continue to serve traffic as load balancers (like the service proxy) remove the Pod from
-   the list of endpoints as soon as the termination grace period _begins_.
+   should not continue to serve regular traffic, unless for the session termination,
+   and should be terminating connection. Without feature
+   gate `EndpointSliceTerminatingCondition` (in beta starting Kubernetes version 1.22),
+   load balancers (like the service proxy) remove the Pod from
+   the list of EndpointSlices as soon as the termination grace period _begins_. With the feature
+   gate `EndpointSliceTerminatingCondition`, EndpointSlices are not removed for the terminating pods
```

Comment:
the list of EndpointSlices as soon as the termination grace period _begins_.

   If your cluster has the default behavior (the EndpointSliceTerminatingCondition
   feature gate is enabled), then EndpointSlices continue to include Pods even when
   those Pods start to terminate,
```